src/Codeception/Util/Annotation.php

```diff
@@ -130,6 +130,9 @@ public function fetchAll(string $annotation): array
         $attr = $this->attribute($annotation);
         if ($attr) {
             if (!$attr->isRepeated()) {
+                if ($annotation === 'example') {
+                    return [$attr->getArguments()];
+                }
```

Comment:
This change broke  tests\cli\RunCest.php:runWithExamples 

Please fix it.

Reply:
I've missed this test, but it is the one that highlights the issue and conflicts with the changes I propose in this PR.

It is confusing that a single `#[Example]` attribute is treated not in the same way as multiple ones.
This also forces you to update attribute arguments when you move from single `#[Example]` to multiple and vice versa.

Like in this flow:
```php
// write simple test - works

#[Examples([1, 1])]
public function example1(AttrsTester $I, Example $e)
{
    $I->assertEquals($e[1], $e[0]);
}

// duplicate line, update data - fails

#[Examples([1, 1])]
#[Examples([2, 2])]
public function example2(AttrsTester $I, Example $e)
{
    $I->assertEquals($e[1], $e[0]);
}

// unwrap arrays to make it work - works

#[Examples(1, 1)]
#[Examples(2, 2)]
public function example3(AttrsTester $I, Example $e)
{
    $I->assertEquals($e[1], $e[0]);
}

// same in the opposite direction
```
I see how this change can introduce backward incompatibility for those who have used a single `#[Example]`. But since the documentation does not describe the specific behavior of a single `#[Example]`, I don't think it's a big issue compared to the improved developer experience.

So I propose to update the `tests\cli\RunCest.php:runWithExamples`.

I'd appreciate your thoughts on this.